.github/workflows/performance-integration-tests.yml

```diff
@@ -0,0 +1,29 @@
+# Merge the yml file to master branch for the cron job schedule to be effective.
+# Reference: https://github.community/t/on-schedule-per-branch/17525
+name: performance-integration-tests
+
+on:
+  # See cron syntax references:
+  #   - https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule
+  #   - https://crontab.guru/
+  schedule:
+    # Runs every 1 hours.
+    # TODO: It is unclear when the timer starts.
+    - cron:  '0 */1 * * *'
+
+jobs:
+
+  # Build and run the Integration Tests for the Firebase performance E2E Test App.
+  performance-integration-tests:
+    # Firebase Performance lives in private repository only currently.
+    # Remove the check after Firebase Performance is open sourced.
+    if: github.repository == 'FirebasePrivate/firebase-ios-sdk'
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: perf
```

Comment:
Might be worth a comment indicating that this is specifying the `perf` branch and should also be removed on open source.

Reply:
Added TODO to describe the reason and future removal, thank you for calling it out!